services/provisioning-service/src/main/java/com/epam/datalab/backendapi/core/response/handlers/ResourcesStatusCallbackHandler.java

```diff
@@ -85,6 +87,16 @@ protected EnvStatusDTO parseOutResponse(JsonNode resultNode, EnvStatusDTO baseSt
         return baseStatus;
     }
 
+    private String CheckAndMapStatus(String status) {
```

Comment:
Rename method according to the Java convention

Reply:
Ty, done.